src/wiseagents/graphdb/lang_chain_wise_agent_graph_db.py

```diff
@@ -104,6 +104,8 @@ def __new__(cls, *args, **kwargs):
         obj._entity_label = "entity"
         obj._neo4j_graph_db = None
         obj._neo4j_vector_db = None
+        obj._properties = None
+        obj._collection_name = None
```

Comment:
I don't think this is the right fix. Both properties and collection_name are required params. So if they're missing from the YAML config for a test we should add these to the YAML config.

Reply:
I've submitted https://github.com/wise-agents/wise-agents/pull/289